docs/snippets/azure/AppHost/Program.ConfigureCosmosInfra.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Azure.Provisioning.CosmosDB;
+
+internal static partial class Program
+{
+    public static void ConfigureCosmosInfra(IDistributedApplicationBuilder builder)
+    {
+        // <configure>
+        builder.AddAzureCosmosDB("cosmos-db")
+            .ConfigureInfrastructure(infra =>
+            {
+                var cosmosDbAccount = infra.GetProvisionableResources()
+                                           .OfType<CosmosDBAccount>()
+                                           .Single();
+
+                cosmosDbAccount.Kind = CosmosDBAccountKind.MongoDB;
+                cosmosDbAccount.ConsistencyPolicy = new()
+                {
+                    DefaultConsistencyLevel = DefaultConsistencyLevel.Strong,
+                };
+                cosmosDbAccount.Cors.Add(new CosmosDBAccountCorsPolicy()
```

Comment:
This feels like a bad sample from a security POV

Reply:
I removed it.